autoscheduler/frontend/src/redux/reducers/courseCards.ts

```diff
@@ -31,8 +31,8 @@ const initialCourseCardArray: CourseCardArray = {
   0: {
     course: '',
     customizationLevel: CustomizationLevel.BASIC,
-    web: false,
-    honors: false,
+    web: 'include',
```

Comment:
Tbh we might as well switch everything to use `online` instead of `web` internally just so we stay consistent. Since this also includes changing the Django models and `scrape_courses` I'm fine with this being in a separate PR though